nginx_config_reloader/__init__.py

```diff
@@ -320,7 +320,8 @@ def process_IN_DELETE(self, event):
             logger.warning("Configuration dir %s not found, waiting..." % dir_to_watch)
             time.sleep(5)
 
-        wm.add_watch(dir_to_watch, pyinotify.ALL_EVENTS, nginx_config_changed_handler, rec=recursive_watch)
+        wm.add_watch(dir_to_watch, pyinotify.ALL_EVENTS, nginx_config_changed_handler,
+                     rec=recursive_watch, auto_add=True)
```

Comment:
for future reference

> auto_add (bool) - Automatically add watches on newly created directories in watched parent |path| directory. If |auto_add| is True, IN_CREATE is ored with |mask| when the watch is added.

http://seb.dbzteam.org/pyinotify/pyinotify.WatchManager-class.html